subprojects/docs/src/docs/release/notes.md

```diff
@@ -53,6 +53,14 @@ The following are the newly deprecated items in this Gradle release. If you have
 ### Example deprecation
 -->
 
+### Change of default Checkstyle configuration directory
+
+Configuring rules consumed by the Checkstyle plugin in a multi-project build used to be tedious.
+Every sub project needs to provide a configuration file even if the content is exactly the same.
```

Comment:
Not really, the user just had to change the file location in the configuration. I don't think people were actually so helpless that they put the same file in all locations.

Reply:
That's exactly what the reporter of the issue said if I understand him correctly: https://github.com/gradle/gradle/issues/2765. He also mentions that the default can be changed but I am not sure if that's what he actually did to work around the issue. I can expand on the description here by saying that you can also set the `configDir` property.

Reply:
Quoting from the issue: 

> Currently every time I set up a multi-project build using Gradle I need to have this boilerplate code overriding configFile to rootProject.

Reply:
Corrected the wording.